bin/options.js

```diff
@@ -21,6 +25,31 @@ module.exports = getOptions;
  */
 const defaultPathname = 'test/mocha.opts';
```

Comment:
Change this to incorporate `getTestDirectory()`.
See [this code](https://github.com/mochajs/mocha/blob/fcb6fb4972f1a9feeb0b3ceff72a6dac906adc3e/bin/options.js) from #3381. Feel free to incorporate...